RedHat-UX/red-hat-design-system
# docs: tokens search tweak

## What I did

Made some visual tweaks and JS changes to the temporary Tokens Search page. These are temporary at least until we can implement the [new design (XD)](https://xd.adobe.com/view/e26f0898-9926-4577-92f0-bb3bb122af9a-b73c/screen/66e862df-bebb-44ae-8a21-f8b0c187574f/).

## Testing Instructions

1. Check out the Deploy Preview
2. Go to Tokens > Overview
3. Search for a term, like `light` (should get results) or `muted` (zero results)
4. Does it look good?
